src/main/java/org/alliancegenome/curation_api/services/helpers/diseaseAnnotations/DiseaseAnnotationCurie.java

```diff
@@ -27,6 +27,10 @@ public static String getConditionRelationUnique(ConditionRelation relation) {
 	}
 
 	public abstract String getCurieID(DiseaseAnnotationDTO annotationDTO);
+	
+	public abstract String getCurieID(GeneDiseaseAnnotation annotation);
+	public abstract String getCurieID(AlleleDiseaseAnnotation annotation);
+	public abstract String getCurieID(AGMDiseaseAnnotation annotation);
```

Comment:
@markquintontulloch curious why you need these three? Does the other function work its doing the same thing... there is a lot of duplicate code here?

Reply:
We need the subject field to generate the curie, and that's not available from the core DiseaseAnnotation class - or am I missing a trick here?

Reply:
Yes thats true but every DiseaseAnnotation has a curie... putting in a PR to this so you can see what I mean.